bot/cogs/remind_cog.py

```diff
@@ -0,0 +1,52 @@
+import logging
+
+import discord.ext.commands as commands
+
+import bot.extensions as ext
+from bot.utils import converters
+from bot.messaging.events import Events
+
+log = logging.getLogger(__name__)
+
+class RemindCog(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @ext.command()
+    @ext.long_help(
+        """
+        This command will direct message the user with their message at specified time in the future.
+
+        This command supports many different time specifiers as listed below.
+        Specifiers:
+            - years: `Y`, `y`, `year`, `years`
+            - months: `M`, `month`, `months`
+            - weeks: `w`, `W`, `week`, `weeks`
+            - days: `d`, `D`, `day`, `days`
+            - hours: `H`, `h`, `hour`, `hours`
+            - minutes: `m`, `min`, `minute`, `minutes`
+            - seconds: `S`, `sec`, `s`, `second`, `seconds`
+
+        """
+    )
+    @ext.short_help('Creates a reminder')
+    @ext.example((
+        'remind 24h',
+        'remind 5m Exam!',
+        'remind 1h Homework!',
+        'remind 4y Graduation',
+        'remind 3y1M4w1d5h9m2s Pi'
+    ))
+    async def remind(self, ctx: commands.Context, wait: converters.Duration, *args):
+
+        await ctx.message.add_reaction('⏰')
```

Comment:
We should respond with an embed confirming the time and message, clembot always reacts with an embed to represent success so we should follow that pattern. This should also be put at the very end just in case adding the reminder throws an exception for some reason so we dont give a false positive